docs/input/docs/usage/examples.md

```diff
@@ -67,6 +86,19 @@ public void Foo(Maybe<Bar> maybeBar = default)
 }
 ```
 
+It's also possible to retrieve the value (or a default value) from the `Maybe` in a method:
+
+```csharp
+private readonly defaultBar = new Bar();
+
+/// maybeBar is an optional parameter. Maybe.None<Bar>() is default,
```

Comment:
`Maybe.None<Bar>() is the default value.`

Reply:
Thx, fixed it.